src/data/WorkshopSchedule.js

```diff
@@ -1,75 +1,186 @@
-import React from 'react';
-
-const hothDate = '2021-02-23'; // replace with real date later
+const hothDate = '2021-01-'; // replace with real date later
```

Comment:
This is the wrong month. Also, it'd be nice to rename this variable to represent more accurately the data it contains.
```suggestion
const hothMonth = '2021-02-';
```